src/modules/identity/utilts.ts

```diff
@@ -0,0 +1,18 @@
+import { AuthIdentity } from 'decentraland-crypto-fetch'
+import { localStorageGetIdentity } from '@dcl/single-sign-on-client'
+
+export function* getIdentityOrRedirect(
```

Comment:
It seems that we're using the `getIdentityOrRedirect` from the sagas, should we keep this function as well?

Reply:
removed!